### PR TITLE
UIBULKED-210: Improve user errors for invalid data - IncorrectTokenCountException error

### PR DESCRIPTION
**Pull request by UladzislauKutarkin**
https://folio-org.atlassian.net/browse/UIBULKED-210
Here made user friendly callout.

https://github.com/folio-org/ui-bulk-edit/assets/72550466/2eb7cff4-7c3b-4932-a695-89b6acca65e9

### CODE REVIEW

File: src/components/BulkEditList/BulkEditListFilters/BulkEditListFilters.js [hunk 1]
```diff
@@ -210,7 +210,7 @@ export const BulkEditListFilters = ({
       setIsFileUploaded(true);
     } catch ({ message }) {
       showCallout({
-        message: <FormattedMessage id="ui-bulk-edit.error.uploadedFile" />,
+        message: <FormattedMessage id="ui-bulk-edit.error.incorrectFormatted" values={{ fileName:fileToUpload.name }} />,
```
Comment (NikitaSedyx) [inline src/components/BulkEditList/BulkEditListFilters/BulkEditListFilters.js:213]: this should be only when message contains IncorrectTokenCountException

Comment (UladzislauKutarkin) [reply src/components/BulkEditList/BulkEditListFilters/BulkEditListFilters.js]: done!

File: src/components/BulkEditList/BulkEditListFilters/BulkEditListFilters.js [hunk 2]
```diff
@@ -21,7 +21,7 @@ import {
   JOB_STATUSES,
   TRANSLATION_SUFFIX,
   EDITING_STEPS,
-  FILTERS,
+  FILTERS, ERRORS,
```
Comment (NikitaSedyx) [inline src/components/BulkEditList/BulkEditListFilters/BulkEditListFilters.js:24]: plz move to the next line - common codestyle

Comment (UladzislauKutarkin) [reply src/components/BulkEditList/BulkEditListFilters/BulkEditListFilters.js]: moved!

**Review by vashjs — APPROVED** [review]

**Pull request merged**